src/components/MessagesList/MessagesList.vue

```diff
@@ -79,6 +79,7 @@ import debounce from 'debounce'
 import { EventBus } from '../../services/EventBus'
 import LoadingPlaceholder from '../LoadingPlaceholder'
 import ChevronDown from 'vue-material-design-icons/ChevronDown'
+import { uniqueId } from 'lodash'
```

Comment:
This breaks uploading files, creating folder and deleting either in the files app